app/models/manageiq/providers/openshift/container_manager.rb

```diff
@@ -54,11 +54,16 @@ def self.raw_connect(hostname, port, options)
   def self.openshift_connect(hostname, port, options)
     require 'openshift_client'
     api_endpoint = raw_api_endpoint(hostname, port)
-    osclient = OpenshiftClient::Client.new(api_endpoint, api_version)
     # TODO: support real authentication using certificates
-    osclient.ssl_options(:verify_ssl => OpenSSL::SSL::VERIFY_NONE)
-    osclient.bearer_token(options[:bearer]) if options[:bearer]
-    osclient
+    ssl_options = {:verify_ssl => OpenSSL::SSL::VERIFY_NONE}
+    auth_options = {}
+    auth_options[:bearer_token] = options[:bearer] if options[:bearer]
```

Comment:
@moolitayer I have the impression that these `auth_options` should be shared in the mixin.